go/cmd/summarize.go

```diff
@@ -32,11 +33,12 @@ func summarizeCmd() *cobra.Command {
 		Example: "vt summarize old.json new.json",
 		Args:    cobra.RangeArgs(1, 2),
 		Run: func(_ *cobra.Command, args []string) {
-			summarize.Run(args, hotMetric)
+			summarize.Run(args, hotMetric, schemaInfoPath)
 		},
 	}
 
 	cmd.Flags().StringVar(&hotMetric, "hot-metric", "total-time", "Metric to determine hot queries (options: usage-count, total-rows-examined, avg-rows-examined, avg-time, total-time)")
+	cmd.Flags().StringVar(&schemaInfoPath, "schema-info-path", "", "Path to output of 'vt schema' command (optional)")
```

Comment:
I thought we said the schema info was going to be another json file that summarize can use as input.

Reply:
I thought we had said it could be optional since we may not have access to the db to get the schema info?